### DISCUSSION TITLE
Need help!

### DISCUSSION THREAD

**Discussion by RunninPigeon** [TrianglyRU/OrbinautFramework#24] [Q&A] [answered]
I'm trying to follow the guide into making a new zone, but whenever i try and access this new zone i get a error message saying something about the camera not being set before reading it, i've following everything correctly too!
![image_2024-09-06_145756164](https://github.com/user-attachments/assets/38ae7b1d-dc8f-4598-b793-34d0459a3b3e)

**Comment by TrianglyRU** [comment]
You'll get this error in a few cases:
- `c_stage` is initialised before `c_framework`, meaning c_stage was added first
- `c_framework` does not exist in the room at all

Please double check everything. If you follow the guide carefully, you won't get any errors. If still not fixed, please attach a screenshot of your room with layer list and objects visible

**Reply by RunninPigeon** [reply]
Got it, now i got a blank grey screen.

**Reply by TrianglyRU** [reply]
I can't really tell what's the issue without having screenshots, code or any other media provided. In that case all I can say is follow the guide

**Reply by RunninPigeon** [reply]
ah sorry bout that, here:![image_2024-09-06_182452385](https://github.com/user-attachments/assets/c143af85-c268-4ef3-acc5-86fc65a52b2f)

**Reply by TrianglyRU** [reply, marked as answer]
I believe you simply do not have  the `c_stage` controller placed in the room if nothing is happening

**Reply by RunninPigeon** [reply]
oh woops, now its fixed!

**Reply by TrianglyRU** [reply]
Please, just double check yourself before looking for help next time. The guide is written as accessible as possible
